content/engine/security/_index.md

```diff
@@ -103,20 +103,22 @@ arbitrary containers.
 
 For this reason, the REST API endpoint (used by the Docker CLI to
 communicate with the Docker daemon) changed in Docker 0.5.2, and now
-uses a UNIX socket instead of a TCP socket bound on 127.0.0.1 (the
+uses a Unix socket instead of a TCP socket bound on 127.0.0.1 (the
```

Comment:
LOL, actually sent me on a search to check what the proper capitalization is 🤔 https://www.linfo.org/unix_upper.html

Reply:
It's a bit of a mess. I've taken to using a proper noun for referring to unix-like. It doesn't really much, but whenever I see the all-caps variant I kind of expect it to refer to the original UNIX. So either Unix or unix in my book.

Reply:
Yeah, Unix is probably fine to use. I  just had in the back of my mind that there was some reason behind the all-caps, but didn't recall the details. https://x.com/thaJeztah/status/1775436628246381011?s=20

Reply:
Updated the wording, hope it's better